app/src/LeadDetailsPage.tsx

```diff
@@ -0,0 +1,58 @@
+import "./LeadDetailsPage.scss";
+import { Lead } from "./Lead.model";
+import axios from "axios";
+import { useParams } from "react-router";
+import { useEffect, useState } from "react";
+
+const server = axios.create({
+  baseURL: "http://localhost:3000",
+});
+
+export function LeadDetailsPage() {
+  const { id } = useParams();
+  const [leadDetails, setLeadDetails] = useState<Lead | null>(null);
+
+  useEffect(() => {
+    async function getDetails() {
+      try {
+        const res = await server.get<Lead>(`http://localhost:3000/leads/${id}`);
```

Comment:
you can "await axios.get..." 
thus removing the defenition of "server" in lines 7-9